src/pages/waiting-room/waiting-room.analytics.effects.ts

```diff
@@ -109,4 +115,23 @@ export class WaitingRoomAnalyticsEffects {
     }),
   );
 
+  @Effect()
+  cbtNumberChanged$ = this.actions$.pipe(
+    ofType(CBT_NUMBER_CHANGED),
+    concatMap(action => of(action).pipe(
+      withLatestFrom(
+        this.store$.pipe(
+          select(getTests),
+        ),
+      ),
+    )),
+    concatMap(([action, tests]: [CbtNumberChanged, TestsModel]) => {
+      this.analytics.logEvent(
+        formatAnalyticsText(AnalyticsEventCategories.WAITING_ROOM, tests),
+        formatAnalyticsText(AnalyticsEvents.CBT_CHANGED, tests),
+        'cbt changed',
```

Comment:
you don't need the label your sending Category: Waiting Room , Event: CBT Chabged, Label: CBT Changed